src/common/words.txt

```diff
@@ -0,0 +1,400 @@
+cheese
```

Comment:
this needs to be removed, i copied/pasted it from somewhere on the internet. we should only add things we actually generated ourselves